views/static/terms.php

```diff
@@ -29,7 +30,7 @@
     </p>
```

Comment:
add the margins back to the terms page

Reply:
You mean the margins that was causing that "cut off issue"?